CMake Errors in googlemock

### description
At commit  880896c the recommended method of including googletest in your project results in cmake errors (see below). I have reproduced this error on FreeBSD 11.2 (cmake 3.12.<something>), Ubuntu 16.04 (cmake 3.5.1), and Ubuntu 18.04 (cmake 3.10.2). 

### fix
Changing cmake to require commit b88511e rather than `master` seems resolve the issue.

```
ExternalProject_Add(googletest
  GIT_REPOSITORY    https://github.com/google/googletest.git
  GIT_TAG           b88511ef640c046ec9c7553b383a2aba64c80967
  SOURCE_DIR        "${CMAKE_BINARY_DIR}/googletest-src"
  BINARY_DIR        "${CMAKE_BINARY_DIR}/googletest-build"
  CONFIGURE_COMMAND ""
  BUILD_COMMAND     ""
  INSTALL_COMMAND   ""
  TEST_COMMAND      ""
)
```

### symptom
```
CMake Error in build/googletest-src/googlemock/CMakeLists.txt:
  Target "gmock_main" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "[...]/build/googletest-src/googletest/include"

  which is prefixed in the build directory.


CMake Error in build/googletest-src/googlemock/CMakeLists.txt:
  Target "gmock_main" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "[...]/build/googletest-src/googletest/include"

  which is prefixed in the build directory.Target "gmock_main"
  INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "[...]/build/googletest-src/googletest/include"

  which is prefixed in the source directory.


CMake Error in build/googletest-src/googlemock/CMakeLists.txt:
  Target "gmock_main" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "[...]/build/googletest-src/googlemock/include"

  which is prefixed in the build directory.


CMake Error in build/googletest-src/googlemock/CMakeLists.txt:
  Target "gmock_main" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "[...]/build/googletest-src/googlemock/include"

  which is prefixed in the build directory.Target "gmock_main"
  INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "[...]/build/googletest-src/googlemock/include"

  which is prefixed in the source directory.
```

--- a/googlemock/README.md
+++ b/googlemock/README.md
@@ -144,27 +144,6 @@ to
     target_link_libraries(example gmock_main)
 
 This works because `gmock_main` library is compiled with Google Test.
-However, it does not automatically add Google Test includes.
-Therefore you will also have to change
-
-    if (CMAKE_VERSION VERSION_LESS 2.8.11)
-      include_directories("${gtest_SOURCE_DIR}/include")
-    endif()
-
-to
-
-    if (CMAKE_VERSION VERSION_LESS 2.8.11)
-      include_directories(BEFORE SYSTEM
-        "${gtest_SOURCE_DIR}/include" "${gmock_SOURCE_DIR}/include")
-    else()
-      target_include_directories(gmock_main SYSTEM BEFORE INTERFACE
-        "${gtest_SOURCE_DIR}/include" "${gmock_SOURCE_DIR}/include")
-    endif()
-
-This will addtionally mark Google Mock includes as system, which will
-silence compiler warnings when compiling your tests using clang with
-`-Wpedantic -Wall -Wextra -Wconversion`.
-
 
 #### Preparing to Build (Unix only) ####
 

